BlogApp/serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from .models import BlogPost, User
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['email','username','password','profile_picture_path','friends']
```

Comment:
I dont think that adding User model to the API is a good idea especially in Social Media project